status-im/status-desktop
# feat(wallet): add support for proxy basic auth for CryptoCompare

### What does the PR do

Add support for proxy basic auth for CryptoCompare

### Affected areas

Wallet market data

### How to test

export BUILD or RUN_TIME env vars and check on server side if credentials are passed.
Tested with localhost http server

### Risk 

Tick **one**:
